documentation/src/components/layout.css

```diff
@@ -55,6 +55,18 @@ governing permissions and limitations under the License.
     margin-right: auto;
 }
 
+@media (max-width: 768px) {
+    #body #layout-content #page {
+        padding: 40px 40px 24px 40px;
+    }
+}
+
+@media (max-width: 568px) {
+    #body #layout-content #page {
+        padding: 40px 26px 24px 26px;
```

Comment:
Just tried to drop the left/right padding to ~75% & 50% of what was there. Without going to odd values.

Reply:
yeah -- realized I should compare to what they're doing. I'll switch to the one breakpoint at a 768 and at least be consistent.